package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "xt",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./bin/www"
```

Comment:
Also add a "build" script which uses ``node`` instead of ``nodemon``. Nodemon should be used only for development.